src/OrbitBase/include/OrbitBase/GetLastError.h

```diff
@@ -9,7 +9,9 @@
 
 namespace orbit_base {
 
+#ifdef _WIN32
```

Comment:
With CMake, this header file is only included in the Windows build, so the ifdef is redundant, was it added because the header makes its way to the Linux Bazel build? Maybe add a comment if that's the case. Another option would be to provide a Linux implementation, something along the lines of : 

```
return strerror(errno);
```

Reply:
It's not actually breaking anything. I just added the ifdef as a precaution, because someone could include this header file and expect this function to be implemented for Linux and this would result in a linker error - which might be surprising.

So I thought it's better to add an ifdef. I expected this to be self-explanatory but if you think it makes it more confusing I can also remove the ifdef again. I don't mind - my primary focus is on the filename change.

Reply:
I was curious to know if there was another reason behind it, apparently not, so totally fine, I guess it documents the fact that there is no Linux implementation yet.